src/Components/WebAssembly/WebAssembly/src/Services/WebAssemblyConsoleLogger.cs

```diff
@@ -21,7 +30,85 @@ public bool IsEnabled(LogLevel logLevel)
         public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception exception, Func<TState, Exception, string> formatter)
         {
             var formattedMessage = formatter(state, exception);
-            Console.WriteLine($"[{logLevel}] {formattedMessage}");
+
+            // If we're given an exception, we want to format it in a useful way, including
+            // inner exceptions and stack traces. The default formatter doesn't do that.
+            if (!(exception is null))
+            {
+                formattedMessage += FormatException(exception);
+            }
```

Comment:
It is a bit weird that the logger does this. Does the ConsoleLogger do the same thing?

Reply:
I would look at what we do in ASP.NET Core and strive for consistency with what is there.

Reply:
See https://github.com/dotnet/extensions/blob/master/src/Logging/Logging.Console/src/ConsoleLogger.cs#L48-L66

Reply:
Good point. Changing as per https://github.com/dotnet/aspnetcore/pull/19606#discussion_r390244378